modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -370,129 +361,132 @@ If you want to update several processes, simply run the command with all the pro
 [NOTE]
 ====
 
-Note: This tool will only change case overview pages. This means that if some of your processes still have process instantiation / task execution forms,
-you need to redesign them in the Studio using Bonita UI designer, as explained in the section above.
+This tool will only change case overview pages. This means that if some of your processes still have process instantiation / task execution forms,
+you need to redesign them in Bonita Studio using Bonita UI Designer, as explained in the section above.
 ====
 
 Example of output issued when running the tool:
 ++++
 <asciinema-player src="_images/images/case_overview_update_mode-ascii.cast" speed="2" theme="monokai" title="Update case overview console output example" cols="240" rows="32"></asciinema-player>
 ++++
 
+=== 6.x application resources
 
-== Migrating to Java 11 in Bonita 7.9
+6.x application resources have been removed in Bonita 7.8.0, so if you are migrating a process that leverages this feature, you need to modify it (for example to use process dependencies instead (Configure > Process dependencies in Bonita Studio)).
 
-Bonita 7.9+ supports Java 11.
+=== Updating to Java 11 in Bonita 7.9 or a greater version
+
+Bonita 7.9 and greater versions support Java 11.
 Migrating an existing platform to Java 11 is not an easy, or painless endeavour.
 To migrate a Bonita platform to Java 11, you need to follow the following steps:
```

Comment:
you need to follow the following steps: => proceed with the following steps:

Reply:
fixed